packages/eds-core-react/src/components/Popover/Popover.tsx

```diff
@@ -199,5 +212,23 @@ export const Popover = forwardRef<HTMLDivElement, PopoverProps>(
         </PopoverPaper>
       </ThemeProvider>
     )
+
+    return (
+      <>
+        {disablePortal ? (
+          popover
```

Comment:
This also needs to be conditionally rendered, and also I think you forgot to remove the css to hide when closed. Also FloatingFocusManager should be possible when no portal